src/io/syncsocket.c

```diff
@@ -259,23 +263,61 @@ struct sockaddr * MVM_io_resolve_host_name(MVMThreadContext *tc, MVMString *host
     return dest;
 }
 
+#define STR_SA_FAMILY(a) a == AF_INET ? "IPv4" : "IPv6"
```

Comment:
I'd write this like so: `#define STR_SA_FAMILY(a) ((a) == AF_INET ? "IPv4" : "IPv6")`